src/Test.UnitTests.BinSkim.Rules/BA2004.EnableSecureSourceCodeHashingTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+
+using Microsoft.CodeAnalysis.BinaryParsers.ProgramDatabase;
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.IL.Rules
+{
+    public class EnableSecureSourceCodeHashingTests
+    {
+        [Fact]
+        public void IsLikelyUwpDummyObjTests()
+        {
+            EnableSecureSourceCodeHashing.IsLikelyUwpDummyObj(Language.MASM, @"c:\dummy.obj", @"c:\dummy.obj").Should().BeTrue();
+            EnableSecureSourceCodeHashing.IsLikelyUwpDummyObj(Language.MASM, @"d:\dummy.obj", @"d:\dummy.obj").Should().BeFalse();
```

Comment:
@shaopeng-gh To make sure I understand this correctly, `d:\dummy.obj` should result in `false`? The drive letter must be `c`?

Reply:
yes I believe the official one should have the path fixed as `c:\dummy.obj`